### DISCUSSION TITLE
Cannot log into Carelink

### DISCUSSION THREAD

**Discussion by kevinfields777** [NightscoutFoundation/xDrip#3405] [Bug reports]
On Match 23rd the Carelink servers went down for maintenance for a few hours. Since then I have not been able to log back in through xDrip. I have tried both my primary Carelink account and follower account. I can log into both from the Carelink app. 

When I pull up system status I'm getting an alert that the token has expired. 
![Screenshot_20240327_162343_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/32113649/9101cf6b-bc35-4356-9573-7a64a025f432)
![Screenshot_20240327_162326_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/32113649/321214c5-269a-4f1c-ab18-3ef0cf7fb42e)

**Comment by Navid200** [comment]
@benceszasz

**Comment by benceszasz** [comment]
@kevinfields777 this seems to be an old version. The latest CareLink Follower version (05.12.2023) has a different status screen, displays more information about the login. It uses a much better login method:
https://github.com/NightscoutFoundation/xDrip/pull/3210

Update your xDrip to a later Nightly version:
https://github.com/NightscoutFoundation/xDrip/releases

**Reply by kevinfields777** [reply]
Yes, I'm running the current stable release. I will try switching to the current Nightly.

**Reply by kevinfields777** [reply]
And it looks like the Nightly from March 27th is now successfully logged in and downloading data. Thank you for your assistance!

**Comment by kevinfields777** [comment]
Well that didn't last long. When I last checked 24 hours ago the data was downloading. But now there is no downloaded data, there is no reporting. It's showing me as logged in but I have no information. 
![Screenshot_20240330_081103_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/32113649/74986abe-586f-43c4-a861-8556b8d54026)
![Screenshot_20240330_081116_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/32113649/9b0091b4-dd5c-44b2-86fb-68c22882edd0)
![Screenshot_20240330_081124_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/32113649/b6200e18-003c-42b1-9c21-4b9b97d9f9f5)
